tests/test_admin/test_django_admin_site.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+from typing import TYPE_CHECKING
+
+import pytest
+from django.contrib import admin
+from django.test import Client, override_settings
+from django.urls import URLPattern, reverse
+
+from tests import factories
+from tests.helpers import patch_method
+from tests.mocks import MockResponse
+from tests.test_external_services.test_verkkokauppa.test_merchant_requests import get_merchant_response
+from tilavarauspalvelu.enums import EmailType
+from tilavarauspalvelu.models import RequestLog, Reservation, SQLLog
+from tilavarauspalvelu.tasks import create_or_update_reservation_statistics
+from tilavarauspalvelu.utils.verkkokauppa.verkkokauppa_api_client import VerkkokauppaAPIClient
+
+if TYPE_CHECKING:
+    from tests.factories._base import GenericDjangoModelFactory
+
+
+@pytest.fixture
+def create_all_models():
+    """Create a model instance for each factory in tests.factories."""
+    for factory_name in factories.__all__:
+        model_factory: GenericDjangoModelFactory = getattr(factories, factory_name)
+
+        if model_factory is factories.ReservableTimeSpanFactory:
+            model_factory.create(
+                start_datetime=datetime.datetime(2024, 1, 1, 12),
+                end_datetime=datetime.datetime(2024, 1, 1, 22),
+            )
+        else:
+            model_factory.create()
+
+        request_log = RequestLog.objects.create(path="foo", body="bar", duration_ms=123)
+        SQLLog.objects.create(
+            request_log=request_log,
+            sql="SELECT * FROM foo",
+            succeeded=True,
+            duration_ns=123,
+            stack_info="stack_info",
+        )
+
+        create_or_update_reservation_statistics(Reservation.objects.values_list("pk", flat=True))
+
+
+@pytest.mark.django_db
+# Override languages, since TinyMCE can't handle lazy language name translations in tests ¯\_(ツ)_/¯
+@override_settings(LANGUAGES=[("fi", "Finnish"), ("en", "English"), ("sv", "Swedish")])
+@patch_method(VerkkokauppaAPIClient.generic, return_value=MockResponse(status_code=200, json=get_merchant_response))
+def test_django_admin_site__pages_load__model_admins(create_all_models):
```

Comment:
Is this test how fast? MInd that if you mark it with `@pytest.mark.slow` then it won't run in CI. Or perhaps we should run slow tests in CI too?

Reply:
I think we disabled them to make tests run faster, but maybe that's a bad idea? Test data generation has been broken before because this is not run.

Reply:
The two new tests take ~3.5sec to run locally for me. Yep I can mark them with the slow flag.

Reply:
But if you mark them with the slow tag, then they won't run in CI. Should we fix this?

Reply:
I'm not sure about running the slow tests in CI too.
On one hand we test more stuff, but then again it will take that much more time to run the CI.
Maybe should be a good thing that we run them here, even if its slower, so we don't necessarily need to always run them locally 🤷

Reply:
Let's test and see how much slower things get, we can always turn them off again

Reply:
To do this, remove `--skip-slow` from [here](https://github.com/City-of-Helsinki/tilavarauspalvelu-core/blob/cd3cbffe6249c82c008037ad5efcc8994a3f8187/.github/workflows/test.yml#L119).